department-of-veterans-affairs/va.gov-cms
# VACMS-5250: Update section labels.

## Description

See #5250.

Needs user guide update and announcement from #5251 before merging. 

This replaces #5151, which was used as a live prototype for testing.

## Testing done

Manual, behat.

## Screenshots
![Content___VA_gov_CMS](https://user-images.githubusercontent.com/643678/117717330-ad986580-b1a8-11eb-8c66-8b506296bfa9.png)

![Edit_CMS_Help_Page__Test_Data__Molestiae_qui_veritatis_ex_in____VA_gov_CMS](https://user-images.githubusercontent.com/643678/117723633-9eb5b100-b1b0-11eb-889f-4d1976c5477d.png)



## QA steps

As admin 

1. Go to https://pr5285-r4jbzvazezkcgutivtgga5l3wjjgi2ek.ci.cms.va.gov/admin/content and see Section labels
2. Go to https://pr5285-r4jbzvazezkcgutivtgga5l3wjjgi2ek.ci.cms.va.gov/node/add, open a smattering of forms, and check out the Section settings. They should all match the above screenshot. 



### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
  - [x] Don't merge yet -- @rachel-kauff is doing this in #5251
